### DISCUSSION TITLE
cnquery: find EC2 instances without owner tags

### DISCUSSION THREAD

**Discussion by misterpantz** [mondoohq/cnquery#556] [Q&A] [answered]
Hi! 👋 I want to find all my AWS EC2 instances that don't have an `owner` tag. With [this help topic](https://mondoo.com/docs/mql/resources/aws-pack/aws.ec2.instance/) I found the `tags` field for `aws.ec2.instance` and am able to list all tags for one or more instances. Yay!! The next step in my challenge is filtering based on the value of a specific tag. Is that possible?

**Comment by arlimus** [comment, marked as answer]
Depending on if you are looking for uppercase or lowercase `owner`:

```coffee
aws.ec2.instances.where(tags['owner'] == null)
```

**Reply by misterpantz** [reply]
Thank you, that works like a charm!
